mgr/src/cmds/nodes.cr

```diff
@@ -1,7 +1,8 @@
 require "./helpers"
+require "json"
 
 struct NodeArgs
-  property status = false, endpoint = ""
+  property status = false, endpoint = "", json = false
```

Comment:
No need to define this with every structs. define in common place. Check the comment above

Reply:
Done